cmd/orb.go

```diff
@@ -56,45 +57,24 @@ func newOrbCommand() *cobra.Command {
 	processCommand.Annotations["PATH"] = orbAnnotations["PATH"]
 
 	publishCommand := &cobra.Command{
-		Use:   "publish",
-		Short: "publish a version of an orb",
-	}
-
-	releaseCommand := &cobra.Command{
-		Use:         "release PATH NAMESPACE ORB SEMVER",
-		Short:       "release a semantic version of an orb",
-		RunE:        releaseOrb,
-		Args:        cobra.ExactArgs(4),
-		Annotations: make(map[string]string),
-	}
-	releaseCommand.Annotations["PATH"] = orbAnnotations["PATH"]
-	releaseCommand.Annotations["NAMESPACE"] = orbAnnotations["NAMESPACE"]
-	releaseCommand.Annotations["ORB"] = orbAnnotations["ORB"]
-	releaseCommand.Annotations["SEMVER"] = "The semantic version used for this release (i.e. 0.3.6)"
-
-	devCommand := &cobra.Command{
-		Use:         "dev PATH NAMESPACE ORB LABEL",
-		Short:       "release a development version of an orb",
-		RunE:        devOrb,
-		Args:        cobra.ExactArgs(4),
+		Use:         "publish <path> <orb>",
```

Comment:
using angle brackets like this is a more standard unix pattern. How do we feel about this?

Reply:
If we like it, I'd like to change all uses of the UPPERCASE forms.